app/containers/pageContainers/Registration.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react';
+import {graphql} from 'react-relay';
+import FormEditUser from '../Forms/FormEditUser';
+import DefaultLayout from '../../layouts/default-layout';
+
+class Registration extends Component {
+  static query = graphql`
+    query RegistrationQuery {
+      query {
+        user(id: "WyJ1c2VycyIsMV0=") {
+          ...FormEditUser_user
+        }
+      }
+    }
+  `;
+
+  render() {
+    const {user} = this.props.query;
+
+    return (
+      <>
```

Comment:
No need for an outer fragment when you have a single child

Reply:
Removed